cpp/tests/interpolate/cubic_spline_test.cu

```diff
@@ -0,0 +1,191 @@
+/*
```

Comment:
Make this a .cpp file since it doesn't contain any CUDA device code. Your compile time will thank you for it.